### PR TITLE
Bra 11 homework 02

### PR DESCRIPTION
**Pull request by dev-kucha**
Homework 11
Part 2

### CODE REVIEW

File: css/stiles.css [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+}
+
+.header__prequel {
+    -webkit-box-sizing: border-box;
+            box-sizing: border-box;
+    color: #767676;
+    font-family: "Proxima Nova", sans-serif;
+    font-size: 15px;
+    font-weight: 400;
+    font-style: normal;
```
Comment (roskyba) [inline css/stiles.css]: Please, remove unnecessary code and rules - `'normal'`s

File: css/stiles.css [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    text-align: left;
+    text-transform: uppercase;
+    letter-spacing: 4.8px;
+    line-height: normal;
```
Comment (roskyba) [inline css/stiles.css]: And here

File: css/stiles.css [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+}
+
+.header__title {
+    -webkit-box-sizing: border-box;
+            box-sizing: border-box;
+    color: #fefefe;
+    font-family: 'Playfair Display', serif;
+    font-size: 48px;
+    font-weight: 400;
+    font-style: normal;
```
Comment (roskyba) [inline css/stiles.css]: And further on

File: css/stiles.css [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+}
+
+.header__title {
+    -webkit-box-sizing: border-box;
+            box-sizing: border-box;
+    color: #fefefe;
+    font-family: 'Playfair Display', serif;
+    font-size: 48px;
+    font-weight: 400;
+    letter-spacing: normal;
```
Comment (roskyba) [inline css/stiles.css]: Not all normal rules were removed. Check carefully, please

File: css/stiles.css [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+    -webkit-box-shadow: -4px -4px 59px rgba(0, 0, 0, 0.16);
+            box-shadow: -4px -4px 59px rgba(0, 0, 0, 0.16);
+    background-color: #fbc02d;
+    padding: 22px;
+}
+
+/* material icons start */
+.material-icons {
+    font-family: 'Material Icons';
+    font-weight: normal;
```
Comment (roskyba) [inline css/stiles.css]: Remove all `normal` in all code

File: css/stiles.css [hunk 6]
```diff
@@ -0,0 +1,195 @@
+/*
```
Comment (roskyba) [inline css/stiles.css]: And rename `stiles.css` ===> `styles.css`

Comment (dev-kucha) [reply css/stiles.css]: Its my special "stile" :-)

**Review by roskyba — APPROVED** [review]

**Pull request closed without merging**